src/styles/blocks/card.scss

```diff
@@ -0,0 +1,69 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-description {
+    color: $color_text--dark;
```

Comment:
Ensure that the variable `$color_text--dark` is defined in your SCSS files. If it's not defined, this will cause a compilation error.